op-e2e/setup.go

```diff
@@ -235,6 +237,10 @@ func (cfg SystemConfig) start() (*System, error) {
 
 	l2Alloc[cfg.L1InfoPredeployAddress] = core.GenesisAccount{Code: common.FromHex(bindings.L1BlockDeployedBin), Balance: common.Big0}
 	l2Alloc[common.HexToAddress(predeploys.L2ToL1MessagePasser)] = core.GenesisAccount{Code: common.FromHex(bindings.L2ToL1MessagePasserDeployedBin), Balance: common.Big0}
+	l2Alloc[common.HexToAddress(predeploys.OVM_GasPriceOracle)] = core.GenesisAccount{Code: common.FromHex(bindings.GasPriceOracleDeployedBin), Balance: common.Big0, Storage: map[common.Hash]common.Hash{
+		// storage for GasPriceOracle to have transctorPath wallet as owner
+		common.BigToHash(big.NewInt(0)): common.HexToHash("0x8A0A996b22B103B500Cd0F20d62dF2Ba3364D295"),
```

Comment:
Is there a way to not hardcode this string and instead reference the wallet that is used?